eslint.config.mjs

```diff
@@ -290,6 +290,12 @@ export default typescript.config([
           message:
             'Since React 19, it is no longer necessary to use forwardRef - refs can be passed as a normal prop',
         },
+        {
+          selector:
+            "CallExpression[callee.object.name='jest'][callee.property.name='mock'][arguments.0.value='sentry/utils/useProjects']",
+          message:
```

Comment:
Appreciate the eslint rule since I assume most of these are just because the person writing the test didn't know they could do this.